src/phy.c

```diff
@@ -141,3 +141,129 @@ static int handle_cca_ed_level(struct nl802154_state *state,
 }
 COMMAND(set, cca_ed_level, "<level>",
 	NL802154_CMD_SET_CCA_ED_LEVEL, 0, CIB_PHY, handle_cca_ed_level, NULL);
+
+static int print_ed_scan_handler(struct nl_msg *msg, void *arg)
+{
+    int r;
+
+    uint8_t status;
+    uint8_t scan_type;
+    uint8_t channel_page;
+    uint32_t unscanned_channels;
+    uint8_t result_list_size;
+    //uint8_t energy_detect_list[ 32 ];
+    char *energy_detect_list;
```

Comment:
this is variable length, and could contain zeros. switch it to a nested attribute, rather than using a string.